awspub/image.py

```diff
@@ -556,6 +557,18 @@ def publish(self) -> None:
                     f"currently using partition {partition}. Ignoring marketplace config."
                 )
 
+        # send ssn notification
+        if self.conf["sns"]:
+            for region in self.image_regions:
```

Comment:
can you move that logic into a helper func `def _sns_publish()` within this file? similar to SSM so in this `publish()` method we only do:
```
if self.conf["sns"]:
    self._sns_publish()
```

Reply:
fixed